packages/iov-bns-governance/src/governor.spec.ts

```diff
@@ -138,18 +138,14 @@ describe("Governor", () => {
   });
 
   describe("getElectionRules", () => {
-    it("throws for non-existent electorateId", async () => {
+    it("returns an empty array for non-existent electorateId", async () => {
```

Comment:
hah! This test discriptions reveals an interesting differentiation. We have two cases to consider:
1. Electorate exists but has no rules
2. Electorate does not exist

For 2. it is fair to throw I think. However, I don't know if we get the data easily without an additional query to the blockchain. Throughts?

Reply:
Yeah, so I think the non-existence of an election rule was being used as a proxy for the non-existence of an electorate to avoid an additional request. I think it's unlikely people will be passing the wrong electorate ID here, and if they do the empty array should be a clue that maybe the electorate doesn't exist.

Reply:
+1
If we consider `electorateId` a filter, it makes perfect sense. And most of all, it fulfils the requirements by the governance app